src/js/action-types.js

```diff
@@ -16,11 +16,35 @@ export const SET_UPLOAD_DATA_TYPES = 'SET_UPLOAD_DATA_TYPES'
 export const SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME = 'SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME'
 export const SET_VISUALIZATION = 'SET_VISUALIZATION'
 
+export const DASHBOARD_CREATE_VISIBILITY = 'DASHBOARD_CREATE_VISIBILITY'
+export const DASHBOARD_CREATE_SUBTITLE_FORM = 'DASHBOARD_CREATE_SUBTITLE_FORM'
```

Comment:
Assuming this is setting the subtitle field in the form, than it should be something like `SET_DASHBOARD_SUBTITLE`. similarly for the title action below.